src/stores/AccountController.ts

```diff
@@ -126,7 +131,21 @@ export async function updateUser(
       throw new Error('no auth');
     }
 
-    const response = await accountService.updateCustomer({ ...values, id: user.id.toString() }, sandbox, auth.jwt);
+    const errors = validateInputLength(values as FirstLastNameInput);
```

Comment:
Should there be a check here to make sure we only call this when values is of type FirstLastNameInput? I imagine we might also want basic validation when the type is EmailConfirmPasswordInput?

Reply:
hmm, I think this part can be completely refactored and improved, the types and validations as well, but I don't think it should be done in this PR.